frontend/src/components/Table/helpers.ts

```diff
@@ -0,0 +1,18 @@
+import { CellProps } from 'react-table';
+import { formatMoney, formatNumber, prettyFormatDate } from 'utils';
+
+/**
+ * These helper methods below provide ways to render common values on a Table cell;
+ *  - dates
+ *  - money
+ *  - percentages
+ */
+
+export const renderDate = <T extends object>({ cell: { value } }: CellProps<T, string>) =>
```

Comment:
Nit: it would help readability to put it under a static class or namespace so it reads **TableUtils.renderDate()** instead of just **renderDate()**

Reply:
I can but that is debatable. The pattern you mention (static classes to hold util methods) is widespread in NET but not javascript. Since every module (file) is kinda of it's own static module there is no need to create an additional static class for that. 

What I mean with this is that every time you "import something from './somemodule'" it gets cached and it is the same thing

Reply:
I see. My comment is only about the readability of the file that is using the util methods. Also, it could avoid namespace collision if using multiple methods with the same name (not sure if that is possible right now).

Reply:
In JS land if you want a "static class" you would do this in the consumer code:

```tsx
import * as TableUtils from '../../helpers';

...

TestUtil.renderDate()
```

Reply:
This is kinda related as to whether to use **React.useEffect()** in code or import **useEffect()** directly. I prefer to use the direct method/helper until a collision is found - then we could namespace it